addons/account/static/src/components/tax_totals/tax_totals.xml

```diff
@@ -7,7 +7,7 @@
                 <label class="o_form_label o_tax_total_label" t-esc="props.taxGroup.tax_group_name"/>
             </td>
 
-            <td>
+            <td  class="o_tax_group">
```

Comment:
Don't think you need to create a new class, can't you use `o_tax_group_edit_input` ?

Reply:
No, the bug is still present if I re-use `o_tax_group_edit_input`.